### PR TITLE
new design

### PR DESCRIPTION
**Pull request by derrickdoo**
Updated theme 
- new nav and layout
- updated homepage copy
- mobile styles

### CODE REVIEW

File: 0100/protocol.html [hunk 1]
```diff
@@ -16,208 +16,215 @@
 -->
 
 <!--#include virtual="../includes/header.html" -->
+<!--#include virtual="../includes/top.html" -->
+<div class="content">
```
Comment (gwenshap) [inline 0100/protocol.html:20]: it doesn't look like we are closing this div tag anywhere?

File: 0100/protocol.html [hunk 2]
```diff
@@ -16,208 +16,215 @@
...
+	<!--#include virtual="../includes/nav.html" -->
+	<div class="right">
```
Comment (gwenshap) [inline 0100/protocol.html:22]: couldn't find where we are closing this either?

Comment (derrickdoo) [reply 0100/protocol.html:22]: Both of these get closed in /includes/footer.html. The old /includes/header.html used to leave these tags open and they'd get closed in footer. 

I broke header.html up a little bit so we could have more flexibility with placing page specific stuff on top of .content.

**Comment by gwenshap** [comment]
LGTM

**Pull request merged**